### PR TITLE
Copyright fix (for addAnnotatedType now provide ID to prevent warning PR#3845)

### PR DESCRIPTION
**Pull request by jGauravGupta**
Signed-off-by: Gaurav Gupta <gaurav.gupta.jc@gmail.com>

### CODE REVIEW

File: ext/cdi/jersey-cdi1x-servlet/src/main/java/org/glassfish/jersey/ext/cdi1x/servlet/internal/CdiExternalRequestScopeExtension.java [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (c) 2015, 2018 Oracle and/or its affiliates. All rights reserved.
+ * (Portions) Copyright (c) 2018 Payara Foundation and/or its affiliates.
```
Comment (jansupol) [inline ext/cdi/jersey-cdi1x-servlet/src/main/java/org/glassfish/jersey/ext/cdi1x/servlet/internal/CdiExternalRequestScopeExtension.java]: I am sorry I rerun with the copyright plugin and Portions copyright does not work. Only simple "Copyright works".

Comment (jansupol) [reply ext/cdi/jersey-cdi1x-servlet/src/main/java/org/glassfish/jersey/ext/cdi1x/servlet/internal/CdiExternalRequestScopeExtension.java]: I can change it if you want

Comment (jGauravGupta) [reply ext/cdi/jersey-cdi1x-servlet/src/main/java/org/glassfish/jersey/ext/cdi1x/servlet/internal/CdiExternalRequestScopeExtension.java]: Should I remove the "(Portions)" from copyright header ?

Comment (jansupol) [reply ext/cdi/jersey-cdi1x-servlet/src/main/java/org/glassfish/jersey/ext/cdi1x/servlet/internal/CdiExternalRequestScopeExtension.java]: Yes please. I'm sorry again. Should have tested it earlier.

Comment (jGauravGupta) [reply ext/cdi/jersey-cdi1x-servlet/src/main/java/org/glassfish/jersey/ext/cdi1x/servlet/internal/CdiExternalRequestScopeExtension.java]: It's okay @jansupol, I have updated the header and thanks for the review.

**Pull request merged**